htdocs/societe/class/api_thirdparties.class.php

```diff
@@ -1374,7 +1374,7 @@ public function generateBankAccountDocument($id, $companybankid = null, $model =
 		$result = $this->db->query($sql);
 		if ($result)
 		{
-			if ($result->num_rows == 0) {
+			if ($result && $this->db->num_rows == 0) {
```

Comment:
I think this->db->num_rows is not defined.
Do you mean  this->db->num_rows($result)

Reply:
I was just about to correct it